extensions/kafka-client/runtime/src/main/java/io/quarkus/kafka/client/runtime/KafkaAdminClient.java

```diff
@@ -61,23 +61,23 @@ public Collection<ConsumerGroupDescription> getConsumerGroups() throws Interrupt
                 .values();
     }
 
-    public boolean deleteTopic(String name) {
+    public boolean deleteTopic(final String name) {
         Collection<String> topics = new ArrayList<>();
         topics.add(name);
         DeleteTopicsResult dtr = client.deleteTopics(topics);
         return dtr.topicNameValues() != null;
     }
 
-    public boolean createTopic(KafkaCreateTopicRequest kafkaCreateTopicRq) {
+    public boolean createTopic(final KafkaCreateTopicRequest kafkaCreateTopicRq) {
         var partitions = Optional.ofNullable(kafkaCreateTopicRq.getPartitions()).orElse(1);
         var replications = Optional.ofNullable(kafkaCreateTopicRq.getReplications()).orElse((short) 1);
```

Comment:
If you use the dev service, I don't believe you can use more than 1, as there is a single node. 
So, we may want to disable that option in this case.

Reply:
Can you clarify what you mean by "disabling the option"? 
Are you asking if to remove the default value?

Reply:
Yes, or add a message in the UI.